test/routes/liana.init.error.test.js

```diff
@@ -1,31 +0,0 @@
-const forestExpress = require('../../src');
```

Comment:
Removed to centralize tests around the init method. It was copied & adapted L.34 of the following file